fullstop/src/main/java/org/zalando/stups/fullstop/Fullstop.java

```diff
@@ -29,7 +32,9 @@
 /**
  * @author jbellmann
  */
-@SpringBootApplication
+@Configuration
+@ComponentScan
+@EnableAutoConfiguration
```

Comment:
I would rather use this convenience annotation, too. but unfortunately my IDE is not able to recognize it.  I would like to keep the three annotations, to gain this nice Spring Framework support.